docs/mivot/viewer.rst

```diff
@@ -110,41 +183,52 @@ The annotation schema can also be applied to table rows read outside of the ``Mi
 with the `astropy.io.votable` API:
 
 .. code-block:: python
-    :caption: Accessing the model view of Astropy table rows
 
     votable = parse(path_to_votable)
     table = votable.resources[0].tables[0]
     # init the viewer
     mivot_viewer = MivotViewer(votable, resource_number=0)
     mivot_object = mivot_viewer.dm_instance
```

Comment:
This example may be out of date.  `resource_number` does not seem to be a valid parameter, and it's not clear if `path_to_votable` is intended as a placeholder or used to point to a concrete example.